Makefile

```diff
@@ -129,16 +129,16 @@ status: status-master $(shell for i in $(shell seq 1 $(NODE_COUNT)); do echo "st
 status-master: ## Show status of the master VM.
 	@STATUS_OUT="$$(vagrant status | tail -n+3)"; \
 		if (( $$(echo "$$STATUS_OUT" | wc -l) > 5 )); then \
-			echo "$$STATUS_OUT" | head -n-5; \
+			echo "$$STATUS_OUT" | tail -r | tail -n +6 | tail -r; \
```

Comment:
Oh well, GNU `tail` does not have the `-r` flag, can you check if Mac has `tac`? It does the same as `tail -r`.

Reply:
Mac didn't have tac by default either.  I settled on keeping your original logic the same but recommending Mac devs install the GNU core utils and then use ghead from those coreutils if it exists.  Please take a look!